src/components/TeamCards/index.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from "react";
+import db from "./../../firebaseConfig";
+import MemberCard from "./../MemberCard";
+import { useTranslation } from "react-i18next";
+import { Row } from "antd";
+import "./index.css";
+
+const TeamCards = () => {
+  const [t] = useTranslation();
+  const [dataState, setDataState] = useState([]);
+
+  const fetchData = async () => {
+    const res = await db.collection("team").doc("teamInfo").get();
```

Comment:
@ibrahimaljarad like this, you can use .doc() to get the ID in your profile page PR